### ISSUE TITLE
repo name change to just "netapp"

### ISSUE THREAD

**Issue by thedoubl3j** [ansible-collections/ansible_collections_netapp#26]
##### SUMMARY

As this repo now lives in a organization call "ansible-collections", might be worth it just to change the name of the repo to just "netapp" and remove the "ansible_collections" prefix.

**Comment by carchi8py** [comment]
I can do this, if that works with you guys,.

I'll need to update our internal pipeline that pushes to Github before making the change.

**Comment by thedoubl3j** [comment]
yeah man, not a problem! should have included a note saying that remotes and stuff would need to get updated.

**Comment by carchi8py** [comment]
The repo has been renamed.
